### PR TITLE
1163 update cache invalidation messages

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/types/types.ts [hunk 1]
```diff
@@ -562,3 +562,14 @@ export type SubscriptionActions = {
     terminate: SubscriptionAction[];
     system: SubscriptionAction[];
 };
+
+export type CacheTag = { type: CacheTagTypes; id?: string };
+
+export enum CacheTagTypes {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/types/types.ts:568]: Enums shouldn't be plural: `CacheTagTypes` -> `CacheTagType`

File: packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts [hunk 2]
```diff
@@ -68,53 +67,25 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
...
-                        const cacheTags: CacheInvalidationTag[] = [cacheTag];
-
-                        if (id) {
-                            cacheTags.push({
-                                type: cacheTag,
-                                id,
-                            });
+                const invalidateTag = (cacheTag: CacheTag) => {
+                    const tagType = (() => {
+                        if (typeof cacheTag === 'object') {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts:72]: According to the type definition in types.ts it is always an object, therefore this check is redundant.

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/types/types.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/types/types.ts:568]: Fixed

File: packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts:72]: fixed

**Review by Georgi2704 — APPROVED** [review]

**Pull request merged**